docker-compose.yaml

```
@@ -103,6 +103,49 @@ services:
       retries: 5
     volumes:
       - agent-ssh-dir:/home/jenkins/.ssh:ro  # Mounts the agent-ssh-dir volume to the /home/jenkins/.ssh path inside the container as read-only
+
+
+  multi_jenkins_controller:
+    image: ${DOCKERHUB_USERNAME}/jenkinsci-tutorials:multi_controller_end-to-end-multibranch
```

```suggestion
    image: ${DOCKERHUB_USERNAME}/jenkinsci-tutorials:multi_controller_
```

I just want to ensure, do we need the branch? (Especially after merging)

---

We won't need it after merging, but it's needed right now,
Should I remove it?

---

I guess so! 👍